### ISSUE TITLE
Keyboard navigation doesn't work in GUI

### ISSUE THREAD

**Issue by jidanni** [BaseXdb/basex#293]
TAB should jump from one field to the other like in Firefox. Also jump into menus and SHIFT TAB also ... all like in Firefox, etc.
The four arrow keys should work to move around in already pulled down menus, like in Firefox.
